packages/realm-server/server.ts

```diff
@@ -77,6 +109,14 @@ export function createRealmServer(realms: Realm[]) {
         return;
       }
 
+      // this one is unique in that it is requested in a manner that is relative
+      // to the URL in the address bar as opposed to the absolute asset location
+      if (realm.paths.local(reversedResolution) === 'worker.js') {
```

Comment:
does this have to be a relative reference? can we serve it like all the other assets?

Reply:
turns out the service worker scope is tied to the path the service worker is served from, in that the scope is always a subset of the path it's served from. If we serve worker.js from /assets, that would mean the service worker scope could only be `/assets` or lower as opposed to `/` which it currently is. so it looks like we can't change this.